docs/sources/next/javascript-api/jslib/http-instrumentation-pyroscope/_index.md

```diff
@@ -0,0 +1,57 @@
+---
+title: "http-instrumentation-pyroscope"
+description: "k6 Pyroscope instrumentation API"
+weight: 04
+---
+
+# HTTP instrumentation for Pyroscope
```

Comment:
The title (line 2) should match the page title (line 7). The suggestion below matches both lines. I've also added menuTitle, which sets how the page title appears in the table of contents. Using menuTitle keeps the page title display matching the other page titles in the k6 docs table of contents. What do you think? 

```suggestion
title: HTTP instrumentation for Pyroscope
menuTitle: http-instrumentation-pyroscope
description: "k6 Pyroscope instrumentation API"
weight: 04
---

# HTTP instrumentation for Pyroscope
```